manual/vscode-extensions/vscode-ember-cli/vscode-ember-cli.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>vscode-ember-cli</id>
+    <title>Ember Cli in Visual Studio Code </title>
+    <version>1.0.0</version>
+    <authors>Felix Rieseberg</authors>
+    <owners>gep13 chocolatey</owners>
```

Comment:
Owners should be a comma separated list as per [.nuspec Reference](https://docs.nuget.org/ndocs/schema/nuspec)